client/src/api/http.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const apiPort = 3001;
+
+export const http = axios.create({
+  baseURL: `http://${window.location.hostname}:${apiPort}`
```

Comment:
What do you think about using `window.location.origin` instead of `.hostname`? this way, if the protocol changes to `https`, no further changes will be required.

Reply:
This will become:
```
baseURL: `${window.location.origin}:${apiPort}`
```

Reply:
@robert-damoc 
You would still have to parse the URL since origin uses the entire URL (including the port). ```
hash: ""
host: "localhost:3000"
hostname: "localhost"
href: "http://localhost:3000/"
origin: "http://localhost:3000"
pathname: "/"
port: "3000"
protocol: "http:"
```
We can do something like ${window.location.protocol}://${window.location.hostname}:{$API_ENV}

Reply:
Yeah, you are right. Using the protocol would be the way to go IMO.